### PR TITLE
Add resolution information to the camera model

### PR DESCRIPTION
**Pull request by PetervDooren**
Noticed that when I deserialised an rgbd file the fullresolution parameter of the camera model was empty

### CODE REVIEW

File: rgbd/src/serialization.cpp [hunk 1]
```diff
@@ -278,6 +276,15 @@ bool deserialize(tue::serialization::InputArchive& a, Image& image)
         return false;
     }
 
+    if (cam_type == CAMERA_MODEL_PINHOLE)
+    {
+        // now that the image size is known add it to the camera model
```
Comment (MatthijsBurgh) [inline rgbd/src/serialization.cpp:281]: This suggest that somewhere earlier in the code there is a comment that we don't know the resolution yet. That comment is missing.

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
See comment about comment ;)

**Comment by MatthijsBurgh** [comment]
The [camera_info](http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/CameraInfo.html) message, does contain, the width and heigth. So I would like to figure out we could integrate that here. Or that we don't have that info, when (de)serializing the message in the current flow.

**Comment by PetervDooren** [comment]
The problem here is (I think). That there are multiple ways in which the image can be encoded. So you cannot create a flow where this info is extracted before the rgb is processed (or depth, depends on which one you use, assumed now is that they use the same camera model.)
And while the camera_info message does contain this information, the cam_info message is used as an intermediary in this process, so it does not contain information that we did not put in

**Comment by MatthijsBurgh** [comment]
Indeed, it contains only the info that we put in.

The question is: Do we not have the info when serializing it, so we can't put it in. Or do we have the info, but we just neglect to add it..

**Comment by PetervDooren** [comment]
The info is added, just not near the other camera info if I recall

**Comment by MatthijsBurgh** [comment]
As discussed, we could look into adding the info on more places.

**Comment by PetervDooren** [comment]
The discussed method is applied in #45

**Pull request closed without merging**